### ISSUE TITLE
cifar10をcnnで学習する

### ISSUE THREAD

**Issue by aiorhiroki** [NCC-AI/Deeper-House-Wife#2]
## 課題3: mnistと同様にcifar10でも高い精度で学習を行うCNNを作る！

`from keras.datasets import cifar10`
から初めて、mnistでやったように、全て初めから自分で書いてみてください。
- 精度はmnistほど高くならないので、頑張ってみてください！
- dropout, batchnormalizationを入れて、精度が向上したことを示してください。

- 注意点は、チャンネルがRGBの3チャンネルです。

**Comment by RemiTakano** [comment]
忙しいところありがとう！やってみるね。
